packages/serverless-helpers/src/configuration/esbuildConfig.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * the recommended esbuild configuration for Swarmion
+ */
+export const recommendedEsbuildConfig = {
```

Comment:
We should export it with a type.
We can create our own, based on https://github.com/floydspace/serverless-esbuild/blob/master/src/types.ts#L24.

Reply:
We can also use some parts of https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda_nodejs.BundlingOptions.html